src/demo/win.html

```diff
@@ -1,87 +1,134 @@
 <html>
-<head>
-    <meta charset="UTF-8">
+  <head>
```

Comment:
The unrelated reformatting of this file is done by the pre-commit hook, not sure if I should try to avoid that..